addons/medical_feedback/RscInGameUI.hpp

```diff
@@ -0,0 +1,14 @@
+
+class RscPictureKeepAspect;
+class RscInGameUI {
+    class RscStanceInfo {
+        controls[] += {QGVAR(BloodVolumeInfoIndicator)};
+        class GVAR(BloodVolumeInfoIndicator): RscPictureKeepAspect {
+            onLoad = QUOTE(uiNamespace setVariable [ARR_2(QQGVAR(BloodVolumeInfoIndicator),_this select 0)]);
```

Comment:
Can also be written as `QUOTE(with uiNamespace do {GVAR(bloodVolumeIndicator) = _this select 0});`. Avoids the need for `ARR_`.

Reply:
Like it better (at least can read :D). Changed.